Src/pch.h

```diff
@@ -218,11 +224,16 @@
 
 #pragma warning(push)
 #pragma warning(disable : 4467 5038 5204 5220)
+#ifdef __MINGW32__
+#include <wrl/client.h>
+#else
 #include <wrl.h>
+#endif
 #pragma warning(pop)
 
 #include <wincodec.h>
 
+#ifndef __MINGW32__
```

Comment:
Many of the XAudio2 headers do not properly protect against multiple definition for non-MSVC-like compilers. Working on getting this fixed, but need this workaround for now.